### PR TITLE
[#2881][managerment job type]

### PR DESCRIPTION
**Pull request by trancongtu-framgia**

### CODE REVIEW

File: resources/assets/admin/css/admin_style.css [hunk 1]
```diff
@@ -1,3 +1,10 @@
 .username-padding {
     padding-left: 10px;
 }
+.form-control, .form-control[readonly] {
+	width: 688px;
```
Comment (BunnyPi04) [inline resources/assets/admin/css/admin_style.css]: tab indent = 4 spaces

File: resources/views/admin/job_types/partials/form.blade.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<div class="col-md-12">
+    <div class="form-group {{ $errors->has('name') ? 'has-error' : '' }}">
+        <div class="form-group">
+            {!! Html::decode(Form::label('name', __('Job Type Name').' <span class="red">*</span>', ['class' => 'control-label col-md-3 col-sm-3 col-xs-12'])) !!}
```
Comment (BunnyPi04) [inline resources/views/admin/job_types/partials/form.blade.php]: convention

File: resources/views/admin/job_types/partials/form.blade.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            <div class="col-md-6 col-sm-6 col-xs-12">
+                {{ Form::text('name', null, ['class' => 'form-control', 'maxlength' => 100]) }}
+                {!! $errors->first('name', '<span class="help-block">:message</span>') !!}
+            </div>
+        </div>
+    </div>
+    <div class="form-group {{ $errors->has('') ? 'has-error' : '' }}">
+        <div class="form-group">
+            {!! Html::decode(Form::label('description', __('Description').' <span class="red">*</span>', ['class' => 'control-label col-md-3 col-sm-3 col-xs-12'])) !!}
```
Comment (BunnyPi04) [inline resources/views/admin/job_types/partials/form.blade.php]: convention

File: resources/views/elements/button_model.blade.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+                        </div>
+                        <div class="modal-footer">
+                            <div class="row">
+                                <div class="col-md-6 text-right">
+                                    {{ Form::open([ 'method' =>'delete', 'url' => route($nameRoute.'.destroy', $data->id) ]) }}
+                                        <button type="submit" class="btn btn-default">Yes</button>
+                                    {{ Form::close() }}
+                                </div>
+                                <div class="col-md-6 text-left">
+                                    <button type="button" class="btn btn-default" data-dismiss="modal">No</button>
```
Comment (BunnyPi04) [inline resources/views/elements/button_model.blade.php]: sử dụng laravel collective

File: resources/views/elements/button_model.blade.php [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+            <div id="myModal" class="modal fade" role="dialog">
+                <div class="modal-dialog">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <h4 class="modal-title">{{ __('Bắt đầu nhập / Có đồng ý không') }}</h4>
+                        </div>
+                        <div class="modal-footer">
+                            <div class="row">
+                                <div class="col-md-6 text-right">
+                                    {{ Form::open([ 'method' =>'delete', 'url' => route($nameRoute.'.destroy', $data->id) ]) }}
```
Comment (BunnyPi04) [inline resources/views/elements/button_model.blade.php]: convention

File: routes/web.php [hunk 6]
```diff
@@ -12,5 +12,6 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('clients.index');
```
Comment (BunnyPi04) [inline routes/web.php]: ko return trong route, đưa vào controller

**Comment by trancongtu-framgia** [comment]
ready

**Comment by trancongtu-framgia** [comment]
ready

**Pull request merged**